test/functional/commands/apps/Create.Tests.ps1

```diff
@@ -0,0 +1,24 @@
+Describe "apps create" {
+  It "creates a new app when all required parameters are provided with valid values" {
+    # Arrange
+    $previousNumberOfApps = (appcenter apps list --output json | ConvertFrom-Json).Count
+    $appDisplayName = "TestAppsCreate$((Get-Date).ToString("yyyy-MM-dd_HH.mm.ffffff"))"
+    $appOs = "Android"
+    $appPlatform = "Java"
+
+    # Act
+    $app = appcenter apps create --platform $appPlatform --os $appOs --display-name $appDisplayName --output json | ConvertFrom-Json
+
+    # Assert
+    $app.displayName | Should -Be $appDisplayName
+    $app.os | Should -Be $appOs
+    $app.platform | Should -Be $appPlatform
+
+    $newNumberOfApps = (appcenter apps list --output json | ConvertFrom-Json).Count
+    $newNumberOfApps | Should -Be ($previousNumberOfApps + 1)
+
+    # Aftermath
```

Comment:
'Aftermath' -> 'Cleanup'? Can we do that like AfterEach or something like that?

Reply:
Doesn't start with A! ^^
I'll structure the tests better with before(each) and after(each) code once more tests appear (next week).